keepassxc-browser/options/options.html

```diff
@@ -599,8 +607,8 @@ <h2 class="tab-header" data-i18n="optionsAboutTab"></h2>
                 </div>
               </div>
 
-              <div class="card shadow">
-                <div class="card-header">
+              <div class="card my-4 shadow">
```

Comment:
add 1.5rem space - i hope that is OK. looks the same as the space to the sidebar.